server/__init__.py

```diff
@@ -35,7 +35,7 @@ def load(info):
     resource = DockerResource('slicer_cli')
     info['apiRoot'].slicer_cli = resource
 
-    dockerImageModel = ModelImporter.model('dockerimagemodel', 'slicer_cli')
+    dockerImageModel = ModelImporter.model('docker_image_model', 'slicer_cli')
```

Comment:
I'd recommend just using the name `docker_image`, by convention we do not need to include "model" in the name of models.